charts/lfx-v2-query-service/templates/deployment.yaml

```diff
@@ -28,11 +28,11 @@ spec:
             - name: PAGE_TOKEN_SECRET
               valueFrom:
                 secretKeyRef:
-                  name: query-svc-secrets
+                  name: lfx-v2-query-service-secrets
```

Comment:
Since this is configurable, it should use the value passed:
```
{{ .Values.secret.name }}
```
This is fine if we don't expect any other secrets, otherwise it should be namespaced like:
```
{{ .Values.secret.pageToken.name }}
```